.envrc

```diff
@@ -0,0 +1 @@
+use flake .#full
```

Comment:
No objection, but confirming this is intentional

Reply:
It was not

Reply:
switching between my old branch which has it committed deletes my uncommitted file on master branch